webpack/components/AnsibleHostDetail/components/JobsTab/PreviousJobsTable.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import { translate as __ } from 'foremanReact/common/I18n';
+
+import RelativeDateTime from 'foremanReact/components/common/dates/RelativeDateTime';
+
+import {
+  TableComposable,
+  Thead,
+  Tbody,
+  Tr,
+  Th,
+  Td,
+} from '@patternfly/react-table';
+
+import { decodeId } from '../../../../globalIdHelper';
+import withLoading from '../../../withLoading';
+
+const PreviousJobsTable = props => {
+  const columns = [
+    __('Description'),
+    __('Result'),
+    __('State'),
+    __('Executed at'),
+    __('Schedule'),
+  ];
+
+  return (
+    <React.Fragment>
+      <h3>Previously executed jobs</h3>
+      <TableComposable variant="compact">
+        <Thead>
+          <Tr>
+            {columns.map(col => (
+              <Th key={col}>{col}</Th>
+            ))}
+          </Tr>
+        </Thead>
+        <Tbody>
+          {props.jobs.map(job => (
+            <Tr key={job.id}>
+              <Td>
+                <Link to={`/job_invocations/${decodeId(job.id)}`}>
```

Comment:
didn't test this, but pretty sure it doesnt work.

Reply:
Correct, now that hash router is on the page, we need https://github.com/theforeman/foreman/pull/8769